packages/webdriver/src/request.js

```diff
@@ -93,6 +93,15 @@ export default class WebDriverRequest extends EventEmitter {
                 return resolve(body)
             }
 
+            /**
+             *  stop retrying as this will never be successful.
+             *  we will handle this at the elementErrorHandler
+             */
+            if(error.message.includes("stale element reference")) {
```

Comment:
Don't we have reference to exceptions like the StaleElementReferenceException somewhere?
Would be nice to have one place for reference and be able to do: instance of StaleElementReferenceException
Though I can imagine it being outside of the scope of this pull request, I think it's worth to mention ;)

Reply:
This was in [v4](https://github.com/webdriverio-boneyard/v4/blob/master/lib/helpers/constants.js#L33-L73). I was thinking of eventually creating an module to handle and reference all kinds of WebDriver related errors but this is out of scope of this PR.